content/docs/sample.rst

```diff
@@ -171,8 +173,11 @@ The YAML data should be structured as follows:
       name: 'Sara Taillon'
       url: 'http://orcasinc.com'
```

Comment:
I wonder if this should link to our CoachHub case study instead, like don't send the user away to another website so quickly, especially without warning.

Reply:
Which link are you actually looking at? We do still want to keep this URL in the YAML data, and there are some places ("view project") that we certainly want to use it. Are you talking about which link we use in the quotes?

Reply:
For now, I'm updating the link from a quote-attribution to go to the case-study if it exists, otherwise the `/open-source/` or `/work/` index, depending on the type of project.

Reply:
This made sense to me in theory, but actually playing with it I find it confusing for the attribution to link somewhere local. (Why should "Founder at Lab06" take me to the generic `/work/` page?) I'd say if the quote has a `url` we link to that, but otherwise quotes don't need to link anywhere.